addOns/oast/src/main/resources/org/zaproxy/addon/oast/resources/Messages.properties

```diff
@@ -0,0 +1,31 @@
+# This file defines the default (English) variants of all the internationalised messages.
```

Comment:
I believe per the latest CrowdIn work this leadin line should be removed along with the alignment/padding in the name value pairs.